src/index_emails.py

```diff
@@ -166,22 +180,24 @@ def load_from_file():
     create_index()
 
     if tornado.options.options.skip:
-        logging.info("Skipping first %d messages from mbox file" % tornado.options.options.skip)
+        logging.info("Skipping first %d messages" % tornado.options.options.skip)
 
-    count = 0
     upload_data = list()
-    logging.info("Starting import from file %s" % tornado.options.options.infile)
-    # mbox = mailbox.UnixMailbox(open(tornado.options.options.infile, 'rb'), email.message_from_file)
 
-    # removed the above UnixMailbox which is not supported in python 3.x and replaced it with mailbox.mbox class
-    mbox = mailbox.mbox(tornado.options.options.infile)
+    if tornado.options.options.infile:
+        logging.info("Starting import from mbox file %s" % tornado.options.options.infile)
+        mbox = mailbox.mbox(tornado.options.options.infile)
+    else:
+        logging.info("Starting import from MH directory %s" % tornado.options.options.indir)
+        mbox = mailbox.MH(tornado.options.options.indir, factory=None, create=False)
 
-    emailParser = DelegatingEmailParser([AmazonEmailParser(), SteamEmailParser()])
+    #emailParser = DelegatingEmailParser([AmazonEmailParser(), SteamEmailParser()])
```

Comment:
Can you bring this back plz?

Reply:
Done.

Pylint complained about the unused variable, so I cleaned it up while in the vicinity.

Reply:
Good point.
You know, let's just remove all the email parser stuff, don't think that's really relevant any longer. Do you mind removing the three files and few lines of code from this file ?

Reply:
Sure. Files and their usages are removed now.